proxy/InkAPITest.cc

```diff
@@ -321,12 +323,15 @@ client_handler(TSCont contp, TSEvent event, void *data)
     // Fix me: how to deal with server side cont?
     TSContDestroy(contp);
     return 1;
-  } else {
+  } else if (TS_EVENT_NET_CONNECT == event) {
     sockaddr const *addr = TSNetVConnRemoteAddrGet(static_cast<TSVConn>(data));
     uint16_t input_server_port = ats_ip_port_host_order(addr);
 
     sleep(1); // XXX this sleep ensures the server end gets the accept event.
```

Comment:
I think the sleep should be after the `write` because the point is to give the kernel time to trigger the `accept`. I'll make that change as I merge this.